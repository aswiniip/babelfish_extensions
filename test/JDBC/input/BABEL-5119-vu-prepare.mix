-- tsql
CREATE DATABASE BABEL5119_1
GO

USE BABEL5119_1
GO
CREATE TABLE t1(a int)
INSERT INTO t1 VALUES(1)
GO
CREATE VIEW v1 AS SELECT 1
GO
CREATE PROCEDURE p1 AS SELECT 1
GO

CREATE LOGIN login_babel5119_1 WITH PASSWORD = '12345678'
CREATE LOGIN login_babel5119_2 WITH PASSWORD = '12345678'
ALTER authorization on database::BABEL5119_1 TO login_babel5119_1
GO
-- terminate-tsql-conn