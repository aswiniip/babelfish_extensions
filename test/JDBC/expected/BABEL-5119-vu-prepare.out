-- tsql
CREATE DATABASE BABEL5119_1
GO

USE BABEL5119_1
GO
CREATE TABLE BABEL5119_t1(a int)
GO
CREATE TABLE BABEL5119_t2(a int)
GO
CREATE TABLE BABEL5119_t3(a int)
GO
INSERT INTO BABEL5119_t1 VALUES(1)
GO
~~ROW COUNT: 1~~

CREATE VIEW BABEL5119_v1 AS SELECT 1
GO
CREATE VIEW BABEL5119_v2 AS SELECT 1
GO
CREATE VIEW BABEL5119_v3 AS SELECT 1
GO
CREATE PROCEDURE BABEL5119_p1 AS SELECT 1
GO
CREATE PROCEDURE BABEL5119_p2 AS SELECT 1
GO
CREATE PROCEDURE BABEL5119_p3 AS SELECT 1
GO

CREATE LOGIN login_babel5119_1 WITH PASSWORD = '12345678'
CREATE LOGIN login_babel5119_2 WITH PASSWORD = '12345678'
ALTER authorization on database::BABEL5119_1 TO login_babel5119_1
GO
-- terminate-tsql-conn
